Development.md

```diff
@@ -0,0 +1,273 @@
+# Convox Rack Development Guide
```

Comment:
Do we say anywhere what is this all about? something like "Rack is a private deployment system that would enable app developer to ... It is built on AWS and Docker and is designed with industry best practices OOTB", etc.